src/shell/atuin.bash

```diff
@@ -34,6 +35,6 @@ fi
 
 if [[ -z ${ATUIN_NOBIND} ]]; then
     bind -x '"\C-r": __atuin_history'
-    bind -x '"\e[A": __atuin_history'
-    bind -x '"\eOA": __atuin_history'
+    bind -x '"\e[A": __atuin_history --filter-mode session'
```

Comment:
I'm a bit against changing defaults at this point, every time we've done it we've had a bunch of people complain that things are different

I've actually had a lot of feedback that global-on-up is something people really like

Either way - I'd be totally happy to highlight that this is configurable in our docs, call it out in the release etc